client/src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { Menu, Button, Icon } from 'semantic-ui-react';
+
+export default class Navbar extends Component {
+  state = {
+    companyName: 'Tripen',
+  };
+
+  render() {
+    return (
+      <Menu>
+        <Menu.Item header>{this.state.companyName}</Menu.Item>
+        <Menu.Menu position="right">
+          <Menu.Item>
+            <Button primary>
+              <Icon name="plus"></Icon>
+              Create Timeline
+            </Button>
+          </Menu.Item>
+
```

Comment:
Why new line?

Reply:
This is a part of my Prettier settings. Shall I change it?